### ISSUE TITLE
Wrong Argument

### ISSUE THREAD

**Issue by ellimondo** [edalzell/statamic-mailchimp#37]
Don't shoot!

Updated to 2.0.3 and the form is now talking to the app, but submitting the form logs the following error:

```
[2020-07-30 16:16:13] local.ERROR: Argument 1 passed to Edalzell\Mailchimp\Listeners\AddFromSubmission::handle() must be an instance of Statamic\Forms\Submission, instance of Statamic\Events\SubmissionCreated given 
```

**Comment by edalzell** [comment]
Oh my tests passed, nice find!

**Comment by edalzell** [comment]
fixed in 2.0.4
